internal/cmd/grants.go

```diff
@@ -174,6 +180,12 @@ func removeGrant(cli *CLI, cmdOptions grantsCmdOptions) error {
 
 	user, group, err := checkUserGroup(client, cmdOptions.Name, cmdOptions.IsGroup)
 	if err != nil {
+		var cliError Error
+		if errors.As(err, &cliError) {
+			return Error{
+				Message: fmt.Sprintf("Cannot revoke grants: %s", cliError.Message),
```

Comment:
I don't think I understand why we only wrap the error with additional context if it's already a `cmd.Error`. Wouldn't we want to add this context regardless of the underlying error?